File: benchmark/prime.lua
function isprime(n)
    local nn = n - 1
    for i = 2, nn do
        if (n % i == 0) then
            return false
        end
    end
    return true
end


function primes(n)
    local count = 0

    for i = 2, n do
        if (isprime(i)) then
            count = count + 1
        end
    end
    return count
end

local N = 10 * 10000

print(primes(N))

File: benchmark/prime.py
def isprime(n):
    nn = n - 1
    for i in xrange(2, nn):
        if n % i == 0:
            return False
    return True


def primes(n):
    count = 0

    for i in xrange(2, n):
        if isprime(i):
            count = count + 1
    return count


N = 10 * 10000

print(primes(N))
